test/parallel/test-cli-permission-deny-fs.js

```diff
@@ -126,3 +129,19 @@ const fs = require('fs');
   assert.strictEqual(status, 1);
   assert.ok(!fs.existsSync('permission-deny-example.md'));
 }
+
+{
+  const firstPath = path.sep + process.cwd().split(path.sep, 2)[1];
```

Comment:
Would the test still work if `firstPath` is `etc`?

Reply:
No. As expected. Unless in some of our machines, the Node.js test happens under the`/etc/` path I think we are good.

Reply:
I don't think that's an OK assumption because we don't know where people might have the project checked out locally. At a minimum we should skip the test in that case.

Reply:
Fair enough. Will do it.